docs/metrics/extending-the-sdk/Program.cs

```diff
@@ -29,10 +29,10 @@ public static void Main(string[] args)
     {
         using var meterProvider = Sdk.CreateMeterProviderBuilder()
             .AddSource("MyCompany.MyProduct.MyLibrary")
-            .AddMetricReader(new MyReader())
+            .AddReader(new MyReader())
```

Comment:
👍 💯

Reply:
It looks like right now you can only add a single reader. But is the intention to remove this limitation at some point? The reason I ask is, it should probably be `SetReader` (like `SetSampler`) if we are only allowing one reader.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#metricreader

_The SDK MUST support multiple MetricReader instances to be registered on the same MeterProvider, and the MetricReader.Collect invocation on one MetricReader instance SHOULD NOT introduce side-effects to other MetricReader instances._

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/issues/2398